src/explore-education-statistics-common/src/modules/charts/types/dataSet.ts

```diff
@@ -50,6 +51,7 @@ export interface DataSetConfigurationOptions {
   colour: string;
   symbol?: ChartSymbol;
   lineStyle?: LineStyle;
+  inlinePosition?: LegendInlinePosition;
```

Comment:
As mentioned earlier - this config is deprecated and doesn't need to have this field